webpack.config.js

```diff
@@ -3,7 +3,7 @@
  * anywhere without any dependencies. It packages the SDK
  * with reqwest and Q. It also preserves license comments.
  */
-
+/* global __dirname */
```

Comment:
stale comment.

Reply:
our eslint is setup for client-side. I did not want to set node: true for the whole folder so just a single case here